<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesignWindow</class>
 <widget class="QMainWindow" name="DesignWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1288</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <item row="0" column="3">
     <widget class="HxTableView" name="tbvModelParams">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>22</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>22</number>
      </attribute>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="HxTableView" name="tbvLotParams">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>22</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>22</number>
      </attribute>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2" colspan="2">
     <widget class="HxTableView" name="tbvBlocks">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>22</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>22</number>
      </attribute>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="HxTableView" name="tbvDesign">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>201</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::DoubleClicked|QAbstractItemView::EditTrigger::EditKeyPressed</set>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>22</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>22</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionLoad"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNew">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Assets/Icons/icons.qrc">
     <normaloff>:/plus_48px.png</normaloff>:/plus_48px.png</iconset>
   </property>
   <property name="text">
    <string>Thêm</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Assets/Icons/icons.qrc">
     <normaloff>:/close_window_48px.png</normaloff>:/close_window_48px.png</iconset>
   </property>
   <property name="text">
    <string>Xóa</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../Assets/Icons/icons.qrc">
     <normaloff>:/save_48px.png</normaloff>:/save_48px.png</iconset>
   </property>
   <property name="text">
    <string>Lưu</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="../Assets/Icons/icons.qrc">
     <normaloff>:/reset_48px.png</normaloff>:/reset_48px.png</iconset>
   </property>
   <property name="text">
    <string>Tải lại</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HxTableView</class>
   <extends>QTableView</extends>
   <header>HxTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Assets/Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
